qemu/tests/cfg/driver_load.cfg

```diff
@@ -2,35 +2,133 @@
     type = driver_load
     kill_vm_on_error = yes
     login_timeout = 240
-    driver_load_cmd = modprobe DRIVER_ID
-    driver_unload_cmd = modprobe -r DRIVER_ID
     repeats = 1
     variants:
-        - nic_device:
+        - with_nic:
             no e1000
             nics += " nic2"
-            nic_model_nic2 = e1000
-            driver_id_cmd = lsmod |grep virtio_net
-            driver_id_pattern = virtio_net
-            test_after_load = file_transfer
-            filesize = 100
-            transfer_timeout = 1000
+            nic_model_nic2 = rtl8139
+            nic_model_nic1 = virtio
+            Linux:
+                driver_load_cmd = modprobe DRIVER_ID
+                driver_unload_cmd = modprobe -r DRIVER_ID
+                driver_id_cmd = lsmod |grep virtio_net
+                driver_id_pattern = virtio_net
+                test_after_load = file_transfer
+                filesize = 100
+                transfer_timeout = 1000
             Windows:
                 # The DevCon utility is a command-line utility that acts as an
                 # alternative to Device Manager. Please set up your windows guest
                 # following: http://support.microsoft.com/kb/311272/en-us
                 driver_id_cmd = C:\devcon.exe find * | find "VirtIO Ethernet Adapter"
                 driver_id_pattern = "(.*?):"
-                driver_load_cmd = C:\devcon.exe enable DRIVER_ID
-                driver_unload_cmd = C:\devcon.exe disable DRIVER_ID
-        - balloon_device:
+                driver_load_cmd = C:\devcon.exe enable @DRIVER_ID
+                driver_unload_cmd = C:\devcon.exe disable @DRIVER_ID
```

Comment:
Hi guys, please split your guest os releated configuration to guest-os.cfg.

Thanks,
Xu